packages/client/src/declarations/index.ts

```diff
@@ -2075,3 +2075,8 @@ export function filterProcessingDeclarationsFromQuery(
     )
   }
 }
+
+export function getDeclarationEvent(event: string): Event {
+  if (['Death', 'DEATH', 'death'].includes(event)) return Event.Death
```

Comment:
This might be a bit more robust for different variations:

```suggestion
  if (event.toLowerCase() === 'death') {
    return Event.Death
  }
```